{"avatar":"//lain.bgm.tv/pic/user/m/icon.jpg","floor":"#1","group":"～技术宅真可怕～","groupHref":"/group/a","groupThumb":"//lain.bgm.tv/pic/icon/m/000/00/00/11.jpg","message":"sudo -s  <br>apt-get update<br>apt-get install python-pip<br><br>pip install shadowsocks<br>vi /etc/shadowsocks.json<br>{<br>    \"server\":\"0.0.0.0\",<br>    \"port_password\": {<br>        \"11383\": \"密码1\",<br>        \"8848\": \"密码2\"<br>    },<br>    \"timeout\":300,<br>    \"method\":\"aes-256-cfb\",<br>    \"fast_open\": false<br>}<br><br><br><br>vi config.json<br><br>    {<br><br>    “server”:”.0.0.0.0.”,<br><br>    “server_port”:*****,<br><br>    “local_port”:*****,<br><br>    “password”:”*****”,<br><br>    “timeout”:300,<br><br>    “method”:”aes-256-cfb”<br><br><br>最后 输入ssserver -c /etc/shadowsocks.json -d start的时候<br>root@ip-172-31-20-248:~# ssserver -c /rtc/shadowcocks.json -d start<br>INFO: loading config from /rtc/shadowcocks.json<br>Traceback (most recent call last):<br>  File \"/usr/local/bin/ssserver\", line 9, in <module><br>    load_entry_point('shadowsocks==2.8.2', 'console_scripts', 'ssserver')()<br>  File \"/usr/local/lib/python2.7/dist-packages/shadowsocks/server.py\", line 34, in main<br>    config = shell.get_config(False)<br>  File \"/usr/local/lib/python2.7/dist-packages/shadowsocks/shell.py\", line 149, in get_config<br>    with open(config_path, 'rb') as f:<br>IOError: [Errno 2] No such file or directory: '/rtc/shadowcocks.json'<br>  这是咋回事……………… ","time":"2016-8-22 18:22","title":"亚马逊上AWS搭建ShadowSocks的正确姿势 - -搭建失败啦","userId":"301953","userName":"我只来看看","userSign":""}